test/user.js

```diff
@@ -0,0 +1,45 @@
+var assert = require('chai').assert;
+var expect = require('chai').expect;
+
+const database = require('../db/database');
+const user = require('../db/user');
+
+describe('saving records', function() {
+    it('Establish a connection with database', function(done) {
+        database.connect();
+        done();
+    });
+
+    it('Sign up a user', function(done) {
+        var newUser = new user.User({
+            id: "aaa",
+            password: "111"
+        });
+
+        newUser.save().then(res => {
+            expect(res.id).to.equal("aaa");
+            expect(res.password).to.equal("111");
+            done();
+        }).catch(err => {
+            done(new Error("Failed to sign up user" + err));
+        });
+    });
+
+    it('Sign in a user', function(done) {
+        var findUser = new user.User({
+            id: "aaa",
+            password: "aaa"
+        });
+
+        if(expect(user.User.findOne(findUser.id))){
```

Comment:
please give spacing between { and after if.
Look at other if cases I wrote. change every line in this test case
e.g.
```
if (expect(user.User.findOne(findUser.id))) {
}
```